### PR TITLE
Feat: cmd edge storage objects list

### PR DESCRIPTION
**Pull request by maxwelbm**
Edge storage objects list command

### CODE REVIEW

File: messages/edge_storage/messages.go [hunk 1]
```diff
@@ -1,41 +1,58 @@
...
 
-	SHORT_DESCRIPTION               = "Manages Edge Storage buckets and objects directly through the command line"
-	SHORT_DESCRIPTION_CREATE_BUCKET = "Creates a bucket in Edge Storage"
-	SHORT_DESCRIPTION_LIST_BUCKET   = "List the buckets in Edge Storage"
-	SHORT_DESCRIPTION_DELETE_BUCKET = "Deleted the bucket in Edge Storage"
+	SHORT_DESCRIPTION                = "Manages Edge Storage buckets and objects directly through the command line"
+	SHORT_DESCRIPTION_CREATE_BUCKET  = "Creates a bucket in Edge Storage"
+	SHORT_DESCRIPTION_LIST_BUCKET    = "List the buckets in Edge Storage"
+	SHORT_DESCRIPTION_LIST_OBJECT    = "List the objects in Edge Storage"
+	SHORT_DESCRIPTION_DELETE_BUCKET  = "Deleted the bucket in Edge Storage"
```
Comment (PabloDiehl) [inline messages/edge_storage/messages.go:12]: Shouldn't this be `Deletes the bucket...`, as the SHORT_DESCRIPTION_CREATE_OBJECTS says `Creates...`?

File: messages/edge_storage/messages.go [hunk 2]
```diff
@@ -1,41 +1,58 @@
...
+	SHORT_DESCRIPTION_CREATE_OBJECTS = "Creates a objects in Edge Storage"
```
Comment (PabloDiehl) [inline messages/edge_storage/messages.go:13]: I'd go for

```suggestion
	SHORT_DESCRIPTION_CREATE_OBJECTS = "Creates objects in Edge Storage"
```

or

```suggestion
	SHORT_DESCRIPTION_CREATE_OBJECTS = "Creates an object in Edge Storage"
```

File: messages/edge_storage/messages.go [hunk 3]
```diff
@@ -1,41 +1,58 @@
...
-	LONG_DESCRIPTION               = "Allows users to perform a wide range of operations, from creating and deleting buckets to adding, removing, and manipulating objects."
-	LONG_DESCRIPTION_CREATE_BUCKET = "Allows users to create a bucket in Edge Storage"
-	LONG_DESCRIPTION_LIST_BUCKET   = "Allows users to list their buckets in Edge Storage"
-	LONG_DESCRIPTION_DELETE_BUCKET = "Allows users to delete their buckets in Edge Storage"
+	LONG_DESCRIPTION                = "Allows users to perform a wide range of operations, from creating and deleting buckets to adding, removing, and manipulating objects."
+	LONG_DESCRIPTION_CREATE_BUCKET  = "Allows users to create a bucket in Edge Storage"
+	LONG_DESCRIPTION_LIST_BUCKET    = "Allows users to list their buckets in Edge Storage"
+	LONG_DESCRIPTION_LIST_OBJECT    = "Allows users to list their objects in Edge Storage"
+	LONG_DESCRIPTION_DELETE_BUCKET  = "Allows users to delete their buckets in Edge Storage"
+	LONG_DESCRIPTION_CREATE_OBJECTS = "Allows users to create a objects in Edge Storage"
```
Comment (PabloDiehl) [inline messages/edge_storage/messages.go:20]:
```suggestion
	LONG_DESCRIPTION_CREATE_OBJECTS = "Allows users to create objects in Edge Storage"
```

File: pkg/api/storage/storage.go [hunk 4]
```diff
@@ -60,25 +60,41 @@ func (c *Client) UpdateBucket(ctx context.Context, name string) error {
 	return nil
 }
 
+func (c *Client) CreateObject(ctx context.Context, fileOps *contracts.FileOps, bucketName, objectKey string) error {
+	logger.Debug("Creating object")
+	req := c.apiClient.StorageAPI.StorageApiBucketsObjectsCreate(ctx, bucketName, objectKey).
+		Body(fileOps.FileContent).ContentType(fileOps.MimeType)
+	_, httpResp, err := req.Execute()
+	if err != nil {
+		logger.Debug("Error while creating object to the edge storage", zap.Error(err))
```
Comment (PabloDiehl) [inline pkg/api/storage/storage.go:69]:
```suggestion
		logger.Debug("Error while creating object in the edge storage", zap.Error(err))
```

**Review by PabloDiehl — COMMENTED** [review]
Just a few typos to fix.

**Review by PabloDiehl — DISMISSED** [review]

**Review by PabloDiehl — APPROVED** [review]

**Review by hendriksoares — APPROVED** [review]

**Pull request merged**